frontend/public/components/catalog/catalog-page.jsx

```diff
@@ -21,8 +21,8 @@ import {
   getServiceClassImage,
   getTemplateIcon,
 } from './catalog-item-icon';
-import { ClusterServiceClassModel, ClusterServiceVersionModel, TemplateModel } from '../../models';
-import { providedAPIsFor, referenceForProvidedAPI } from '../operator-lifecycle-manager';
+import { ClusterServiceClassModel, TemplateModel } from '../../models';
+import { ClusterServiceVersionModel, providedAPIsFor, referenceForProvidedAPI } from '@console/operator-lifecycle-manager';
```

Comment:
Really this should be contributed as a new extension point so that core doesn't import from an extension

Reply:
Do you want me to add a `CatalogModel` extension point in this pull request?

Reply:
I think it makes sense if the effort isn't too large. I could see other plugins wanting to contribute to the catalog at some point.

Reply:
@spadgett Added a new commit with the `DevCatalogModel` extension.